src/components/Theme/style.scss

```diff
@@ -349,8 +363,10 @@ $logo-height: 32 * $unit;
     color: var(--osdk-color-content-alert-error);
   }
 
-  &:hover {
-    background-color: var(--osdk-color-background-alert-error-link-hover);
+  @media (hover: hover) {
```

Comment:
I can see that this is not supported on IE11. Will this affect buttons' behaviour on IE11? I doubt it but might be worth testing on IE11.